server/service/src/sync/translations/requisition.rs

```diff
@@ -269,7 +269,7 @@ impl SyncTranslation for RequisitionTranslation {
                 date_and_time_to_datetime(data.date_entered, 0),
                 from_legacy_sent_datetime(data.last_modified_at, &r#type),
                 from_legacy_finalised_datetime(data.last_modified_at, &r#type),
-                data.daysToSupply as f64 / NUMBER_OF_DAYS_IN_A_MONTH,
+                data.daysToSupply as f64 / 30.0,
```

Comment:
Might be nice to have a `MSUPPLY_NUMBER_OF_DAYS_IN_A_MONTH` constant or something?
Feels slightly sad to use a less accurate number just to align with mSupply.